data_scientist_skills/data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import string
+import unidecode
+
+def get_data(df1_dir, df2_dir): #Relative path to each data file
+    #Get data
+    df1 = pd.read_csv(df1_dir)
+    df2 = pd.read_csv(df2_dir)
+    #Drop Columns
+    df1.drop(columns='Unnamed: 0', inplace=True)
+    df2.drop(columns = ['Unnamed: 0', 'index'], inplace = True)
+    #Concat to full dataframe
+    df = pd.concat([df1, df2], ignore_index=True, axis=0)
+
+    return df
+
+def get_description(df):
+    return pd.DataFrame(df['Job Description'])
+
+def clean(description):   # .apply to description or title column
+    stop_words = set(stopwords.words('english'))
```

Comment:
maybe add a doc string here, for good practice?
''' Function returns cleaned text from one input string, applicable for description columns and/or title column'''